### ISSUE TITLE
Dropping InstanceAccelerationStructureDescriptor causes a segfault in autoreleasepool

### ISSUE THREAD

**Issue by K0bin** [gfx-rs/metal-rs#324]
```rust
autoreleasepool(|| { let _ = metal::InstanceAccelerationStructureDescriptor::descriptor(); });
```

crashes with a segfault.

[Repro.zip](https://github.com/gfx-rs/metal-rs/files/15502618/Repro.zip)

**Comment by grovesNL** [comment]
I believe this was fixed with #320, could you retry using `master`?

**Comment by K0bin** [comment]
I worked around it locally by manually sending retain before it goes out of scope. So that merged PR should definitely fix it.
